src/python/orquestra/model.py

```diff
@@ -124,6 +130,8 @@ def nested_arrays_to_lists(obj):
 
   return obj
 
+# Helper function for loading models in JSON format
+# Numpy arrays are the expected type to set the weights of a Keras model
 def nested_lists_to_arrays(obj):
```

Comment:
This function should also have docstrings + should it be moved into a `utils` file? Here or in `zquantum.core`? Will defer to you on if this is something others might use or not